src/lua_details.cc

```diff
@@ -187,6 +143,11 @@ long long get_ms_since_epoch(TimePoint t0, std::chrono::milliseconds dt)
     using std::chrono::milliseconds;
     using std::chrono::round;
 
+    static_assert(std::numeric_limits<long long>::max()
+                  >= std::numeric_limits<TimePoint::rep>::max());
+    static_assert(std::numeric_limits<long long>::max()
+                  >= std::numeric_limits<milliseconds::rep>::max());
+
     const long long t0_ms = round<milliseconds>(t0.time_since_epoch()).count();
```

Comment:
Maybe this is off-topic but can't we use `std::chrono::milliseconds` instead of the general `long long`? For me it is clearer than using `long long` where we only extract from the member function name its semantics. See also member function `check_script_timeout(lua_State*):void` where we use the result.

Reply:
Because the value needs to be stuffed into the Lua registry, and that needs exactly `long long`.

Reply:
The question comes up 'everywhere', I believe we should go away from `long long` (and the `LL` suffix) to some more explicit type name. That also makes it clear for users of the library why what happens.

Reply:
> The question comes up 'everywhere', I believe we should go away from `long long` (and the `LL` suffix) to some more explicit type name. That also makes it clear for users of the library why what happens.

Fair enough. 👍 
But that is a different PR. :)